#imu_topic: "/imu"

use_odom: false
use_local_target: false
debug: false
is_filter_ground: false

map_frame: "/map"
odom_frame: "/odom"
base_frame: "/base_link"
map_topic: "/map/point_cloud"
# odom_topic: "/odom/imu"
odom_topic: "/odomImu/odom_imu"

tf_timeout: 1.0
odom_timeout: 1.5

predict_error_thresh: 0.3

voxel_leaf_size: 0.5
ndt_resolution: 1.0
ndt_max_iterations: 25
ndt_step_size: 0.1
ndt_epsilon: 0.01

if_init_pose_with_param: false
init_x: 0.0
init_y: 0.0
init_z: 0.0
init_roll: 0.0
init_pitch: 0.0
init_yaw: 0.0

min_scan_range: 1.0  
max_scan_range: 90

length_update_target_map: 3.0
target_map_radius: 100
